src/main/scala/ai/privado/threatEngine/CookieConsentMgmtModule.scala

```diff
@@ -1,8 +1,8 @@
-package ai.privado.languageEngine.java.threatEngine
+package ai.privado.threatEngine
 
 import ai.privado.cache.{AppCache, DataFlowCache, RuleCache}
+import ThreatUtility.hasDataElements
```

Comment:
Can we import this from the base package `ai.privado.threatEngine.ThreatUtility.hasDataElements` ?

Reply:
Addressed here: https://github.com/Privado-Inc/privado-core/pull/814/commits/1f896170d831e6bfac21ffbb33348b16be85211c#diff-375232dc9bd151701124e2409d6be0e510254595781008fcc5979995326c9137R4